arches/settings_utils.py

```diff
@@ -40,20 +42,32 @@ def _move_to_end_of_sys_path(*paths, add_cwd=False):
         sys.path = _orig_sys_path
 
 
-def list_arches_app_names():
-    return [
-        config.name
-        for config in apps.get_app_configs()
-        if getattr(config, "is_arches_application", False)
-    ]
+def get_app_configs_from_module(module_name):
+    module = import_module(module_name)
```

Comment:
Would be nice to get away from the settings mechanics and thus not have to do this.